python/setup.py

```diff
@@ -146,7 +146,6 @@ def get_packages(self):
 
 # NOTE: The lists below must be kept in sync with ray/BUILD.bazel.
 ray_files = [
-    "ray/core/src/ray/thirdparty/redis/src/redis-server" + exe_suffix,
```

Comment:
Safe to remove?

Reply:
Sorry, my bad. I'll put it back... it's left from the previous PR.